README.md

```diff
@@ -242,6 +242,16 @@ The newly created bundles are located in checkbox/umd and checkbox/cjs folders.
 
 If everything works, add a README.md file to the component folder and add a documentation page to the docs/ folder.
 
+# Adding styles
+
+The styles of a components are located in the same location as its source in a `style.css` file. You can use different file name and folder structure if it makes sense. For example [the styles of the slider component](https://github.com/CoherentLabs/GameUIComponents/tree/master/components/slider/styles) are located in a separate folder and they have names corresponding to the type of slider they style.
+**Make sure to prefix all selectors with guic- to avoid overwriting global styles with common names**. The prefix is an abbreviation of game ui components.

```

Comment:
```suggestion
**Make sure to prefix all selectors with `guic-` to avoid overwriting global selectors with common names**. The prefix is an abbreviation of Game UI Components.
```
or instead of "global selectors" -> "global ones".